accepted/PSR-6-cache-meta.md

```diff
@@ -276,25 +276,59 @@ default control information is all that's possible), and because it's trivial
 for a particular implementation to include as an add-on should it desire to
 do so.
 
-5. People
+5. Errata
+----------
+
+### 5.1 Handling of incorrect DateTime values in expiresAt()
+
+The `CacheItemInterface::expiresAt()` method's `$expiration` parameter is untyped
+in the interface, but in the docblock is specified as `\DateTimeInterface`.  The
+intent is that either a `\DateTime` or `\DateTimeImmutable` object is allowed.
+However, `\DateTimeInterface` and `\DateTimeImmutable` were added in PHP 5.5, and
+the authors chose not to impose a hard syntactic requirement for PHP 5.5 on the
+specification.
+
+Despite that, any value that is not null or an object of `\DateTimeInterface` (either
+`\DateTime` or `\DateTimeImmutable`, or possibly others added to the language in
+the future) MUST be treated as an invalid syntax error.  Implementers are
+encouraged to actively reject values that do not implement that interface.
+
+Simulating a failed type check unfortunately varies between PHP versions and thus is not
+recommended.  Instead, implementors SHOULD throw an instance of `\Psr\Cache\InvalidArgumentException`.  
+The following sample code is recommended in order to enforce the type check on the expiresAt()
+method:
+
+```php
+
+class ExpiresAtInvalidParameterException implements Psr\Cache\InvalidArgumentException {}
+
+// ...
+
+if (!(null === $expiration || $expiration instanceof \DateTime || $expiration instanceof \DateTimeInterface)) {
+  $error = sprintf('Argument 1 passed to %s::expiresAt() must be an instance of DateTime or DateTimeImmutable, %s given', get_class($this), gettype($expiration));
+  throw new ExpiresAtInvalidParameterException($error);
+}
```

Comment:
Can we please make this more readable, e.g., by splitting the conditional over multiple lines and splitting the `sprintf` over multiple lines? The line length right now makes understanding more difficult.

Additionally, because `$expiration` is un-typed, we should likely test for objects vs. PHP built-ins when generating the error message.

Suggestion:

```php
if (! (
        null === $expiration
        || $expiration instanceof \DateTime
        || $expiration instanceof \DateTimeInterface
)) {
    throw new ExpiresAtInvalidParameterException(sprintf(
        'Argument 1 passed to %s::expiresAt() must be an instance of DateTime or DateTimeImmutable; %s given',
        get_class($this),
        is_object($expiration) ? get_class($expiration) : gettype($expiration)
    ));
}
```